### DISCUSSION TITLE
Pyrosim and SDS: simulating a tyre rubber fire

### DISCUSSION THREAD

**Discussion by LeoPrt** [firemodels/fds#13198] [General]
Hello everyone,
In a first time, i'm sorry for my english. I'm new on the topic, hope I write in the good discussion.

For the context, I'm at an engineering school in France, studying risk management. My team and I have to work on a RETEX, and we chose the fire at a Michelin site (Cataroux) in 2000. It was a rubber fire (used to produce tyres).

We collected some datas and now we wanted to simulate the fire. We wanted to use FDS, but it's difficult. However, we have managed to obtain a licence to use the Pyrosim software. It's more easier to see what we do.

I create some rooms with oak fire to familiarise myself to the software, I used the tutorial available on the Pyrosim site. 
But now, I want to create the same room (just 4 walls, a celling and a door) and use the caracteristic of rubber gum to create a fire. I didn't success to create a correct simulation, it doesn't work. I tried to enter some materials and reactions but I don't understand what I'm doing and it doesn't work.

I'd like to know if it's possible to use a similar material in the library to make my fire easier?
If this isn't possible, what are the steps involved in creating my simulation using the characteristics of the rubber?I can't find any videos or websites that explain this clearly. 

We don't want anything complex. Keep it as simple as possible at first, then go into more detail if possible.

Thank you in advance if anyone takes the time to help me, I'd be very grateful.

**Comment by drjfloyd** [comment]
This forum does not provide support for the commerical product Pyrosim.
Contact Thunderhead Engineering.
